### PR TITLE
Reword section of state channels developer doc

### PR DESCRIPTION
**Pull request by gorondan**

### CODE REVIEW

File: src/content/developers/docs/scaling/state-channels/index.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ After initializing the channel's state, peers interact by signing transactions a
 
 - The transaction which triggers the state transition (e.g., Alice sends 5 ETH to Bob)
 
-State updates in the channel are not broadcasted on-chain as is normally the case when users interact on Mainnet, which aligns with state channels' goal to minimize on-chain footprint. So long as participants agree on state updates, they are as final as an Ethereum transaction. Participants only need to depend on Mainnet's consensus if a dispute comes up.
+State updates in the channel are not broadcasted on-chain as is normally the case when users interact on Mainnet, which aligns with state channels' goal to minimize on-chain footprint. So, as long as participants agree on state updates, they are as final as an Ethereum transaction. Participants only need to depend on Mainnet's consensus if a dispute comes up.
```
Comment (minimalsm) [inline src/content/developers/docs/scaling/state-channels/index.md:77]:
```suggestion
State updates in the channel are not broadcasted on-chain as is normally the case when users interact on Mainnet, which aligns with state channels' goal to minimize on-chain footprint. As long as participants agree on state updates, they are as final as an Ethereum transaction. Participants only need to depend on Mainnet's consensus if a dispute arises.
```

**Review by minimalsm — APPROVED** [review]
Thanks @gorondan for the PR ❤️ The original wasn't a typo, though on reading it back I realise that it uses very colloquial English that non-native speakers or folks from certain regions might not understand. I've amended this so that it is hopefully clearer but keeps the original meaning.

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @gorondan for docs

**Comment by gorondan** [comment]
glad to be able to start contributing with an accepted pull request ❤️, even if a trivial one. will join discord community and looking forward to contributing to the project from now on.

**Pull request merged**